packages/mcp-core/src/tools/search-events/config.ts

```diff
@@ -105,6 +105,31 @@ When user asks for calculated metrics, ratios, or conversions:
 - IMPORTANT: Equations are ONLY supported in the spans dataset, NOT in errors or logs
 - IMPORTANT: When sorting by equations, use "-equation|..." for descending order (highest values first)
 
+PERFORMANCE INVESTIGATION STRATEGY:
+When users ask about "performance problems", "slow pages", "slow endpoints", "latency issues",
```

Comment:
We may want to extend this to include mobile questions in the future, TTID, ANR.

@noahsmartin might have a strong opinion as to what the right ones are.

I also wonder about other performance domains, e.g., query performance.

I believe starting with page / vitals is a good starting point though.

Reply:
it actually has an endpoint it can query - so it should be able to figure these out